app/src/test/java/tech/ula/utils/DeviceDimensionsTest.kt

```diff
@@ -1,5 +1,6 @@
 package tech.ula.utils
```

Comment:
I'd like to see a test or two that ensured correct behavior from `getNavigationBarSize`